### PR TITLE
docs/styled-mode

### PR DESCRIPTION
**Pull request by goransle**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208315061591931

### CODE REVIEW

File: docs/getting-started/your-first-chart.md [hunk 1]
```diff
@@ -67,6 +67,8 @@ We will start off by creating a simple bar chart.
         <script type="text/javascript" src="/js/themes/gray.js"></script>
     ```
 
+    It is also possible to style the chart with CSS using [styled mode](https://www.highcharts.com/docs/chart-design-and-style/style-by-css).
```
Comment (MarkusBarstad) [inline docs/getting-started/your-first-chart.md:70]: Just wondering, should this be point number 5 🤔?

**Review by MarkusBarstad — COMMENTED** [review]
Looks good! 🚀 Only one thing I am uncertain about

File: docs/getting-started/your-first-chart.md (hunk 1 above)
Comment (goransle) [reply docs/getting-started/your-first-chart.md:70]: I was thinking of as an extension of step 4 (therefore the indentation).  Maybe it should be rewritten as a sublist 🤔

Comment (MarkusBarstad) [reply docs/getting-started/your-first-chart.md:70]: Alright, both options make sense to me @goransle , feel free to rerequest review either way!

**Review by MarkusBarstad — APPROVED** [review]
🚀

File: docs/chart-design-and-style/design-and-style.md [hunk 2]
```diff
@@ -176,4 +175,4 @@ In Highcharts, animation is enabled or disabled in two separate places.
 The animation can either be set as a boolean or a configuration object. If true, it will use the "swing" jQuery easing and a duration of 500 ms. If used as a configuration object, the following properties are supported:
 
 *   duration - The duration of the animation in milliseconds.
-*   easing - When using jQuery as the general framework, the easing can be set to linear or swing. More easing functions are available with the use of jQuery plug-ins, most notably the jQuery UI suite. See the [jQuery docs](https://api.jquery.com/animate/). When using MooTools as the general framework, use the property name transition instead of easing.
\ No newline at end of file
+*   easing - When using jQuery as the general framework, the easing can be set to linear or swing. More easing functions are available with the use of jQuery plug-ins, most notably the jQuery UI suite. See the [jQuery docs](https://api.jquery.com/animate/). When using MooTools as the general framework, use the property name transition instead of easing.
```
Comment (TorsteinHonsi) [inline docs/chart-design-and-style/design-and-style.md:178]: This one is outdated - jQuery and particularly MooTools is history, so we should take this opportunity to fix it.
```suggestion
*   easing - Custom easing can be applied to all animation. See [Easing Functions](https://easings.net/) for a nice overview over some possible functions. Easing functions can either be passed directly, or registered on the `Math` object and referenced by name. See [chart/animation-easing](https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/highcharts/chart/animation-easing) for a live demo. The default easing is `easeInOutSine`.
```

File: docs/chart-design-and-style/style-by-css.md [hunk 3]
```diff
@@ -11,15 +11,33 @@ To customize your styles, you can create your own themes, or just add your own i
 
 Highcharts since v11 honors the [prefers-color-scheme](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme) CSS media feature. If you end user prefers dark mode, the Highcharts CSS will pick this up and set the CSS color variables accordingly. To avoid this, you can set either the `.highcharts-light` or `.highcharts-dark` class name on the chart container. See a [live demo for switching between modes](https://www.highcharts.com/samples/highcharts/css/prefers-color-scheme).
 
+## Loading from node_modules
```
Comment (TorsteinHonsi) [inline docs/chart-design-and-style/style-by-css.md:14]: This header comes out stronger than the simpler option, which is loading from CDN. The CDN is not mentioned specifically, but linked to in the intro. Reorganize a bit? Maybe a header `## Loading the CSS` then with two sub-headers `Loading from CDN` and `Loading from node_modules`?

**Review by TorsteinHonsi — CHANGES_REQUESTED** [review]
Thanks!

**Review by TorsteinHonsi — APPROVED** [review]

**Pull request merged**